src/main/java/space/obminyashka/items_exchange/service/impl/AuthServiceImpl.java

```diff
@@ -48,9 +48,9 @@ public Optional<UserLoginResponseDto> createUserLoginResponseDto(String username
 
     @Override
     public boolean logout(String accessToken, String username) {
-        final String token = JwtTokenProvider.resolveToken(accessToken);
+        final String token = JwtTokenService.resolveToken(accessToken);
```

Comment:
Also it's better to use var here, as for me

Reply:
I'll do that next time once I have code changes in the class.